app/ee/omnichannel/containers/OmnichannelStatus.tsx

```diff
@@ -48,7 +49,9 @@ const OmnichannelStatus = memo(({ searching, goQueue, queueSize, inquiryEnabled,
 				right={() => (
 					<View style={styles.omnichannelRightContainer}>
 						{inquiryEnabled ? <UnreadBadge style={styles.queueIcon} unread={queueSize} /> : null}
-						<Switch value={status} trackColor={SWITCH_TRACK_COLOR} onValueChange={toggleLivechat} />
+						<Touch theme={theme} onPress={toggleLivechat}>
```

Comment:
Wouldn't it be nice to put a hitslop of at least 5?

Reply:
@GleidsonDaniel didn't work only the hitSlop in Switch. Looks like the Touchable of `List.Item` is overwriting the touch of `Switch`

Reply:
For iOS the way was coded in the official version and in this PR is working fine. However, for Android, this PR is fixing the behavior, because the user wasn't able to "turn off" his availability.

So, yes, I tested for both platform this PR 😁